src/App.tsx

```diff
@@ -15,71 +15,94 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type Good = string;
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+type State = {
+  selectedGood: Good;
+  isSelected: boolean;
+};
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export class App extends React.Component<{}, State> {
+  state: State = {
+    selectedGood: 'Jam',
+    isSelected: true,
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  handleSelectClick = (good: string) => {
+    this.setState({
+      isSelected: true,
+      selectedGood: good,
+    });
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  handleRemoveClick = () => {
+    this.setState({
+      isSelected: false,
+      selectedGood: '',
+    });
+  };
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  render() {
+    const { selectedGood, isSelected } = this.state;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+    return (
+      <div>
+        <main className="section container">
+          {!isSelected
```

Comment:
In conditional rendering, it's a good tone to check for a true value instead of its denial.
It will make your code easier to read
```suggestion
          {isSelected
```